sormas-ui/src/main/java/de/symeda/sormas/ui/sormastosormas/SormasToSormasOptionsForm.java

```diff
@@ -56,10 +56,24 @@
 
 	public static SormasToSormasOptionsForm forCase(List<String> excludedOrganizationIds) {
 		return new SormasToSormasOptionsForm(
-			excludedOrganizationIds,
-			true,
-			Arrays.asList(SormasToSormasOptionsDto.WITH_ASSOCIATED_CONTACTS, SormasToSormasOptionsDto.WITH_SAMPLES),
-			null);
+				excludedOrganizationIds,
+				true,
+				Arrays.asList(SormasToSormasOptionsDto.WITH_ASSOCIATED_CONTACTS, SormasToSormasOptionsDto.WITH_SAMPLES),
+				(form) -> {
+					FieldHelper.setEnabledWhen(
```

Comment:
similar thing would be needed for sharing events with event participants too.

Also I would change a bit the order of the options on the dialog to have Handover as a first option because almost all other options depend on it.


Also on the backend would be nice to force sending samples and cleanup pseudonymization option when handing over the ownership, for the in case something doesn't work as expected on the IU. 
But this could go into a separate ticket

Reply:
I asked if this has to be applied to events with participants as well (https://github.com/hzi-braunschweig/SORMAS-Project/issues/6684#issuecomment-922676797), but the ticket states that it concerns cases because it affects calculating the case classification, so I'm not sure if it's relevant for events as well.

I changed the order and moved the ownership and pseudonymization options to the top.

I've also added a backend method to ensure the options are correctly matching the ownership selection.